docs/public/_redirects

```diff
@@ -525,11 +525,12 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 ## MUI X
 ## Unlike the store that expect to be hosted under a subfolder,
 ## MUI X is configured to be hosted at the root.
-/static/x/* https://material-ui-x.netlify.app/static/x/:splat 200
-/x/_next/* https://material-ui-x.netlify.app/_next/:splat 200
-/x/* https://material-ui-x.netlify.app/x/:splat 200
-/r/x-* https://material-ui-x.netlify.app/r/x-:splat 200
-/:lang/x/* https://material-ui-x.netlify.app/:lang/x/:splat 200
+/static/x/* https://docs-v6--material-ui-x.netlify.app/static/x/:splat 200
+/static/* https://mui.com/static/:splat 200
+/x/_next/* https://docs-v6--material-ui-x.netlify.app/_next/:splat 200
+/x/* https://docs-v6--material-ui-x.netlify.app/x/:splat 200
+/r/x-* https://docs-v6--material-ui-x.netlify.app/r/x-:splat 200
+/:lang/x/* https://docs-v6--material-ui-x.netlify.app/:lang/x/:splat 200
```

Comment:
Same as the previous question, what's the right docs we should redirect to?

Reply:
Same answer I commented above.